src/components/table/Table.js

```diff
@@ -93,6 +94,11 @@ class Table extends Component {
     }
 
     if (rows && !_.isEqual(prevState.rows, rows)) {
+      this.setState({
```

Comment:
Dup code

Reply:
True, but the check is totally different for both cases so we won't even save any loc's here